onnxruntime/core/providers/cuda/tensor/size.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/tensor/size.h"
+#include "core/providers/cuda/cuda_fwd.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+ONNX_OPERATOR_KERNEL_EX(
+    Size,
+    kOnnxDomain,
+    1,
+    kCudaExecutionProvider,
+    KernelDefBuilder()
+        .OutputMemoryType<OrtMemTypeCPUOutput>(0)
+        .TypeConstraint("T",
```

Comment:
.TypeConstraint("T", DataTypeImpl::AllTensorTypes()), 

?

Reply:
MLFloat16 and BFloat16 missing... 
AllTensorTypes should be fine here.

Reply:
ok